### PR TITLE
Implement new design

### PR DESCRIPTION
**Pull request by kvntch**
Update navigation, headers and added logo based on template.

### CODE REVIEW

**Review by rorlich — CHANGES_REQUESTED** [review]
Please notice & address ESLint warnings (e.g add alt to img)

File: prisma/schema.prisma [hunk 1]
```diff
@@ -1,5 +1,5 @@
 generator client {
-  provider = "prisma-client-js"
```
Comment (rorlich) [inline prisma/schema.prisma:2]: please do not commit any changes in prisma files. these changes are not related to the task.

**Review by rorlich — CHANGES_REQUESTED** [review]

File: prisma/schema.prisma (hunk 1 above)
Comment (kvntch) [reply prisma/schema.prisma:2]: alright copy on this. I just solve the issue in typescript checking. I will remove it.
